docs/walk-through.md

```diff
@@ -274,11 +316,24 @@ $ entitlementId=$(curl \
     --silent \
     --header "Content-Type: application/json; odata=minimalmetadata" \
     --data "{'applicationId':'$applicationId', 'token':'$AZ_BATCH_SOFTWARE_ENTITLEMENT_TOKEN','duration':'PT5M'}" \
-    $AZ_BATCH_ACCOUNT_URL/softwareEntitlements?api-version=$apiVersion \
+    $AZ_BATCH_ACCOUNT_URL/softwareEntitlements?api-version=9999-09-09.99.99 \
     | jq -re '.entitlementId' || (echo "FAILED: entitlementId not found in JSON response" 1>&2))
+```
```

Comment:
Need to precede these with examples showing the actual response from the server before showing the slice/dice of these as they extract the `entitlementId`.

Reply:
Agree it'd be informative and consistent to include the server response. I wasn't certain how to do so while:
- Only documenting the bulk of the request (`Invoke-RestMethod` or `curl`) once, to reduce clutter and ease maintenance
- Providing a minimal set of commands (as few as possible) for the user to cut and paste (i.e. I'd prefer to avoid two separate commands for storing the server response and parsing out the entitlement id)

How about this approach? I've kept the commands unchanged, but added a description of what the response content looks like (before the `entitlementId` is extracted out).